packages/showcase/src/app/utility-classes/spacing/metadata.ts

```diff
@@ -0,0 +1,78 @@
+import { ComponentMetadata } from "../../../common/types";
+import { ComponentTemplateItemText } from "../../../services/template-language/types";
+
+export const spacingMetadata = {
+  title: "Spacing",
+  description: "",
+  codeSnippets: {
+    blockInline: {
+      linkId: "spacing-body-inline-block",
+      title: "Block Inline",
```

Comment:
It pretty weird this title. It think we should consider a better title.

```suggestion
      title: "Block Inline",
```